strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/registry/support/AuthenticatorsRegistrySerializer.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.authentication.registry.support;
+
+import org.carlspring.strongbox.authentication.api.Authenticator;
+import org.carlspring.strongbox.authentication.registry.AuthenticatorsRegistry;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class AuthenticatorsRegistrySerializer
```

Comment:
I would generally prefer for us to have one way of serializing/deserializing XML.

@sbespalov : Do we need to re-work the `GenericParser`?

@fuss86 : How much of an effort would it be to use the `GenericParser` instead?